backend/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## Unstable features
 - GraphQL endpoints for contracts and modules should currently be seen as unstable, indicating that the query models are likely to undergo changes.
```

Comment:
I think "state of the application" comments like this belong in the readme, not changelog which should be an append-only log.